lab-ronoh/route/note-router.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const noteCrud = require('../lib/note-crud');
+const taskCrud = require('../lib/task-crud');
+const noteRouter = module.exports = new Router();
```

Comment:
Lots of people like this syntax, but I sort of hate it, because if you look, your module.exports is hidden in the middle of a line in the middle of a block of lines. Can you think of a way to make it more obvious?